packages/card/README.md

```diff
@@ -0,0 +1,35 @@
+# Archetype
+
+Mineral UI `Card` component. Find out more about [Mineral UI](https://github.com/mineral-ui/mineral-ui).
+
+
+## Installation
+
+1. Install project [peer dependencies](../../docs/peer-dependencies.md), if not already installed
+
+2. Install the package
+
+  ```sh
+  npm install --save @mineral-ui/card
+  ```
+
+
+## Usage
+
+```jsx
+import React from 'react';
+import { Card, CardTitle, CardBlock } from '@mineral-ui/card';
```

Comment:
I wasn't sure if Card should be a default export or not. This would be our first component without a default.